uint8 output
uint8 OUTPUT_BOTH = 131 #0x83
uint8 OUTPUT_YAW = 129 #0x81
uint8 OUTPUT_YAWRATE = 130 #0x82

uint8 mode
uint8 MODE_STOP = 57 #0x39
uint8 MODE_ONE_TIME = 48 #0x30
uint8 MODE_CONTINUE_20MS = 50 #0x32
uint8 MODE_CONTINUE_50MS = 51 #0x33
uint8 MODE_CONTINUE_100MS = 52 #0x34
uint8 MODE_CONTINUE_200MS = 53 #0x35
uint8 MODE_CONTINUE_250MS = 54 #0x36
uint8 MODE_CONTINUE_500MS = 55 #0x37
uint8 MODE_CONTINUE_1000MS = 56 #0x38

---

int8 result
int8 RESULT_FAIL = -1
int8 RESULT_DONE = 0